virtual_ecosystem/models/hydrology/hydrology_model.py

```diff
@@ -683,6 +777,12 @@ def _update(self, time_index: int, **kwargs: Any) -> None:
                 coords={"cell_id": self.grid.cell_id},
             )
 
+        soil_hydrology["canopy_evaporation"] = self.layer_structure.from_template()
+        soil_hydrology["canopy_evaporation"][
+            self.layer_structure.index_filled_canopy
+            # ] = np.sum(np.array(daily_lists["canopy_evaporation"]), axis=0)
+        ] = np.array(daily_lists["canopy_evaporation"]).sum(axis=(0, 1))
```

Comment:
This returns a sum over all layers, I think it might be beneficial to keep the layer structure intact. Needs to be taken into account in the steps that follow